packages/nouns-webapp/src/pages/Ideas/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { Col, Row } from 'react-bootstrap';
+import Section from '../../layout/Section';
+import Ideas from '../../components/Ideas';
+import { useProposalThreshold } from '../../wrappers/nounsDao';
+import classes from './Ideas.module.css';
+
+const IdeasPage = () => {
+  const threshold = useProposalThreshold();
```

Comment:
lol I don't think we want to use proposal threshold for this. Should talk to team but I feel like no threshold is fine, or maybe just a single lil noun. We want to encourage ideas

Reply:
The scope is to restrict this to 1 lil noun so we get community ideas. This line was copied over from the Proposals page but we can set this to 1 instead of using the same value 👍